extensions/cactus-plugin-htlc-coordinator/src/main/typescript/web-services/withdraw-counterparty-endpoint.ts

```diff
@@ -0,0 +1,102 @@
+import { Express, Request, Response } from "express";
+
+import {
+  IWebServiceEndpoint,
+  IExpressRequestHandler,
+  IEndpointAuthzOptions,
+} from "@hyperledger/cactus-core-api";
+
+import {
+  Logger,
+  Checks,
+  LogLevelDesc,
+  LoggerProvider,
+  IAsyncProvider,
+} from "@hyperledger/cactus-common";
+import { registerWebServiceEndpoint } from "@hyperledger/cactus-core";
+import { WithdrawCounterpartyRequest } from "../generated/openapi/typescript-axios";
+import { PluginHTLCCoordinator } from "../plugin-htlc-coordinator";
+import OAS from "../../json/openapi.json";
+
+export interface IWithdrawCounterpartyOptions {
+  logLevel?: LogLevelDesc;
+  coordinator: PluginHTLCCoordinator;
```

Comment:
@jagpreetsinghsasan Please refactor so that the plugin registry is passed in here instead (the one that the connector plugin has access to)

Reply:
@jagpreetsinghsasan Emphasis on "instead". E.g. remove the coordinator parameter entirely if possible